### DISCUSSION TITLE
Setting Organization to NewsMediaOrganization

### DISCUSSION THREAD

**Discussion by KeldJ** [RehanSaeed/Schema.NET#645] [Q&A] [answered]
How do i set the OrganizationType to NewsMediaOrganization -> https://schema.org/NewsMediaOrganization

**Comment by RehanSaeed** [comment]
Not sure I understand you. There is a type called `NewsMediaOrganization` you can use.

**Reply by KeldJ** [reply]
Im not sure how to set it to NewsMediaOrganization. Organization.Type is readonly

Currently were just doing 

                        Schema.NET.Article document = new Schema.NET.Article();
                        document.Name = title;
                        document.Headline = title;
                        document.DatePublished = System.DateTimeOffset.Now;
                        document.Description = summary;
                        document.Url = new Uri(url);
                        Schema.NET.Person author = new Schema.NET.Person();
                        author.Name = "xxxx";
                        document.Author = author;
                        Schema.NET.Organization organization = new Schema.NET.Organization();
                        organization.Name = "xxxx";
                        if (!string.IsNullOrEmpty(image))
                        {
                            Schema.NET.ImageObject imageObject = new Schema.NET.ImageObject();
                            imageObject.Url = new Uri(image.Contains("http") ? image : "https://" +  image);
                            document.Image = imageObject;
                            
                        }
                        organization.Logo = new Uri("xxxx");
                                                                          
                        document.Publisher = organization;

**Reply by KeldJ** [reply, marked as answer]
Installed Schema.Net.Pending where NewsMediaOrganization are available
